### PR TITLE
Fix Edit Toolbar docs to explain how to use a CreateButton

### PR DESCRIPTION
**Pull request by agent3bood**
https://github.com/marmelab/react-admin/issues/3245

### CODE REVIEW

File: docs/CreateEdit.md [hunk 1]
```diff
@@ -883,6 +883,8 @@ Here are the props received by the `Toolbar` component when passed as the `toolb
 
 **Tip**: To alter the form values before submitting, you should use the `handleSubmit` prop. See [Altering the Form Values before Submitting](#altering-the-form-values-before-submitting) for more information and examples.
 
+**Tip** `<Toolbar>` will pass many props to its children, if you use `<CreateButton>` or other buttons you will get `React does not recognize the <xxx> prop on a DOM element` errors, insteade you can build you own button.
```
Comment (fzaninotto) [inline docs/CreateEdit.md]: There are several things wrong in that change

> `<Toolbar>` will pass many props to its children

Is vague, uses future tense while we try to use present tense everywhere

> if you use `<CreateButton>` or other buttons

Not true: EditButton or DeleteButton don't trigger the error

> insteade

isn't English

> you can build you own button

This isn't clear at all. And besides, the recommendation is rather to wrap an existing button inside another component that will just use the necessary props and drop the rest

Comment (agent3bood) [reply docs/CreateEdit.md]: I have updated my PR, now it is specific and has example

**Comment by fzaninotto** [comment]
Thanks.

**Comment by WiXSL** [comment]
Couldn't this be improve in `CreateButton.tsx` by sanitising this props?

**Pull request merged**